src/main/java/menion/android/whereyougo/utils/Utils.java

```diff
@@ -208,10 +207,9 @@ public static boolean isIntentAvailable(Intent intent) {
         return list.size() > 0;
     }
 
-    /*********************************/
-  /* SECURITY PART */
-
-    /*********************************/
+    /*
+     * SECURITY PART
+    */
```

Comment:
Maybe change to /* SECURITY PART */ like in the other classes?

Reply:
Added in the next force push